axon-mongo-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/mongo/springboot/AxonMongoProperties.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2010-2022. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.extensions.mongo.springboot;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("axon.mongo")
+public class AxonMongoProperties {
+
+    /**
+     * The database name to use for the Axon Framework Mongo Collections.
```

Comment:
Is there a default if this isn't set? Or is this a required field? Bit of a nit of course, but might be good to state that here.

Reply:
Added some more info, it takes it from the factory default, that likely is in the spring mongo config.